x-pack/plugins/observability_solution/infra/public/alerting/common/components/threshold.tsx

```diff
@@ -7,13 +7,13 @@
 
 import React from 'react';
 import { Chart, Metric, Settings } from '@elastic/charts';
-import { EuiIcon, EuiPanel, useEuiBackgroundColor } from '@elastic/eui';
+import { EuiIcon, EuiPanel, type UseEuiTheme, useEuiTheme } from '@elastic/eui';
 import type { PartialTheme, Theme } from '@elastic/charts';
 import { i18n } from '@kbn/i18n';
 import { COMPARATORS } from '@kbn/alerting-comparators';
 
 export interface ChartProps {
-  theme?: PartialTheme;
+  theme?: UseEuiTheme<{}>;
```

Comment:
Would this work? `{}` is the default value of the generic parameter
```suggestion
  theme?: UseEuiTheme;
```

Reply:
I think so, i'll fix it on the next PR

Reply:
We don't use this anywhere but it is meant to be the chart `PartialTheme` which can be used to override values from the `baseTheme`.